src/main/scala/org/constellation/trust/TrustManager.scala

```diff
@@ -95,9 +98,11 @@ object TrustManager {
 
   def calculateScoringMap(scores: List[Id]): Map[Id, Int] =
     scores.sortBy { _.hex }.zipWithIndex.toMap
+  //todo need UUID Int for ID, since when nodes join/leave they will "take" on old score.
+  // Instead, make fixed-size cache of all Full nodes, update when nodes join/leave. Clear the old data too
```

Comment:
@buckysballs it's enough to enable this line. there is no need to make some workarounds for this :)
<img width="938" alt="image" src="https://user-images.githubusercontent.com/4959931/79967102-b380b300-848e-11ea-9d53-7a755b2d4692.png">